build-logic/src/main/java/qsl/internal/analysis/ClassAnalysisUtils.java

```diff
@@ -40,18 +40,25 @@ public static Path getClassPath(FileSystem jarFs, String path) {
 	}
 
 	public static Stream<ClassNode> readPackage(FileSystem jarFs, String targetPackage) throws IOException {
-		return Files.list(jarFs.getPath(targetPackage.replace('.', '/')))
-				.filter(path -> path.toString().endsWith(".class") && Files.isRegularFile(path))
-				.map(path -> {
-					try {
-						return readClass(path);
-					} catch (IOException e) {
-						throw new RuntimeException(e);
-					}
-				})
-				.sorted(Comparator.comparing(classNode -> classNode.name));
+		return readPackage0(Files.list(jarFs.getPath(targetPackage.replace('.', '/'))));
 	}
 
+	public static Stream<ClassNode> readPackageRecursive(FileSystem jarFs, String targetPackage) throws IOException {
+		return readPackage0(Files.walk(jarFs.getPath(targetPackage.replace('.', '/'))));
+	}
+
+	public static Stream<ClassNode> readPackage0(Stream<Path> in) {
```

Comment:
```suggestion
	public static Stream<ClassNode> readPackage(Stream<Path> in) {
```

Come on, we can do better for naming than just suffix it with `0`...